bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -720,8 +720,7 @@ public void asyncAddEntry(final byte[] data, final AddCallback cb,
      * @param ctx
      *            some control object
      */
-    public void asyncAddEntry(final long entryId, final byte[] data, final AddCallback cb, final Object ctx)
-            throws BKException {
+    public void asyncAddEntry(final long entryId, final byte[] data, final AddCallback cb, final Object ctx) {
```

Comment:
The boolean flag would be usefull to tell that the entryid in PendingAddOp is not  initialized, an alternative could be to use a java.lang.Long.
But if we decide to fail all the normal addEntry calls this will no be useful.
@jvrao what do you think? Are you using -1 as vakid entryid? Are you using negative values? Cc @reddycharan